karma.conf.js

```diff
@@ -326,7 +342,7 @@ module.exports = function(config) {
 			singleBundle: false,
 
 			// esbuild options
-			target: 'es2015',
+			target: 'es5',
```

Comment:
This no longer complains about `const` because it is being fed `var` (due to Babel running on all JS files). Setting it to `es5` here just prevents ESbuild from generating ES2015 glue code (for module bindings etc).